### DISCUSSION TITLE
Option for 24h time format

### DISCUSSION THREAD

**Discussion by Tealk** [firefly-iii/firefly-iii#8690] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/explanation/more-information/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been requested before.

### Description

It would be great if there was a 24-hour time format.

![image](https://github.com/firefly-iii/firefly-iii/assets/12276250/449829ba-acfd-424c-9fe9-6545aecb2010)

### Solution

Perhaps the time format could be set accordingly by the language.

### What are alternatives?

Alternatively, there could be a separate option for this.

**Comment by JC5** [comment, marked as answer]
The format is decided by your browser. It's probably set to US.

**Reply by Tealk** [reply]
that's where the information comes from, thank you
but this is a bit inconsistent if you can change the language but not the time format.

**Reply by JC5** [reply]
Yes because I don't have any influence on it. It's your browser.

**Reply by Tealk** [reply]
and if you simply link it to the language you have set?
Or offer a separate option to change it?

**Reply by JC5** [reply]
I cannot change it, how many times must I say it? 😂

I use the "datetime-local" field and it's 100% controlled by the browser, no matter which settings I let you set.

**Reply by Tealk** [reply]
asso sorry, I misunderstood that it simply depends on the browser language, not directly on the browser output.
